dpx/src/dpx_cidtype0.rs

```diff
@@ -2051,10 +2016,11 @@ pub unsafe extern "C" fn CIDFont_type0_t1dofont(mut font: *mut CIDFont) {
         pdf_ref_obj((*font).descriptor),
     );
     let handle = dpx_open_type1_file((*font).ident);
-    if handle.is_null() {
+    if handle.is_none() {
```

Comment:
You can use `except()` here

Reply:
We have many places where `panic!` can be replaced with `assert!`. I did not try to do this here to simplify review.
It is easier to find all panics and handle.